shared/chat/conversation/input-area/normal/index.tsx

```diff
@@ -156,9 +156,6 @@ const useUnsentText = (
   considerDraftRef.current = false
 
   const dispatch = Container.useDispatch()
-  const isExplodingModeLocked = Container.useSelector(state =>
```

Comment:
this is fine but it changes cause we're changing it, so instead just keep track locally so we don't thrash when we go from an empty input to one with input